lib/linguist/heuristics.yml

```diff
@@ -158,7 +158,7 @@ disambiguations:
   - language: Slice
     pattern: '^\s*(#\s*(include|if[n]def|pragma)|module\s+[A-Za-z][_A-Za-z0-9]*)'
```

Comment:
Would it be invalid syntax for a Slice file to start with `{`?

If so, we can replace this heuristic with:

~~~diff
   - language: Slice
-    pattern: '^\s*(#\s*(include|if[n]def|pragma)|module\s+[A-Za-z][_A-Za-z0-9]*)'
+    pattern: '\A(?!\s*[{\[])'
~~~

I suspect the reason why the tests are failing is because Slice files which *don't* contain lines like `#include…` are being left to the classifier.

Reply:
A "proper" Slice file should never start with a `{`. I think it might be possible to contrive an example, but no one should write it like that.

Reply:
That should be enough then. 👍

Reply:
Changed in b213793